invenio_app_ils/ui/src/invenio_app_ils/common/components/ResultsTable/formatters.js

```diff
@@ -6,8 +6,9 @@ import assign from 'lodash/assign';
 import { formatPidTypeToName } from '../ManageRelationsButton/utils';
 import { loan as loanApi } from '../../api';
 import { BackOfficeRoutes, FrontSiteRoutes } from '../../../routes/urls';
-import { invenioConfig } from '../../config';
+import { invenioConfig, TRUNCATE_LENGTH } from '../../config';
```

Comment:
If you do this, then I would say that for backoffice and frontsite it will be different.
I guess it will be better to have a small config for backoffice only then...